vassal-app/src/main/java/VASSAL/counters/PlaySound.java

```diff
@@ -148,23 +148,23 @@ public String getName() {
 
   @Override
   public String getDescription() {
-    return format.getFormat().length() == 0 ? "Play Sound" : "Play Sound - " + format.getFormat();
+    return Resources.getString("Editor.PlaySound.component_type") + ((format.getFormat().length() == 0) ? "" : " - " + format.getFormat());
```

Comment:
I made a new thing for this. Use

return buildDescription("Editor.PlaySound.trait_description", format.getFormat());